crates/ruff/src/rules/flake8_bandit/rules/assert_used.rs

```diff
@@ -23,6 +23,9 @@ use ruff_text_size::Ranged;
 ///
 /// Use instead:
 /// ```python
+/// if not x > 0:
+///     raise ValueError("Expected positive value.")
+/// # or even better:
```

Comment:
Nit: We should probably have an empty line
```suggestion
///     raise ValueError("Expected positive value.")
///
/// # or even better:
```